test/blocks/quiz-entry/quiz-entry.test.js

```diff
@@ -156,4 +156,147 @@ describe('Quiz Entry Component', () => {
     await new Promise((resolve) => setTimeout(resolve, 100));
     expect(continueButton.classList.contains('disabled')).to.be.false;
   });
+  it('should navigate the carousel using keyboard commands', async () => {
+    const options = document.querySelectorAll('.quiz-option');
+    const option = document.querySelector('.quiz-option');
+    option.click();
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
I'm not a big fan of having timeouts in unit tests, as they tend to add up and then the test suite takes a long time to run. I think there was a way around this with `clock.runAllAsync()` or something of the sort. In total this file adds around 2s of wait time. Multiply that by more such files and our PR checks will take a bit too much time to run.